models/honeyswap/honeyswap_v2_trades.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(tags=['dunesql'],
+        schema = 'honeyswap_v2_trades',
```

Comment:
with file name change, can we also change schema to just `honeyswap`?

Reply:
fixed in [e775cd7](https://github.com/duneanalytics/spellbook/pull/4139/commits/e775cd79796092351d629d03eacc65153819a063)